src/app/board-details/board-details.component.ts

```diff
@@ -57,7 +61,7 @@ export class BoardDetailsComponent implements OnInit, OnChanges {
       .getEndBoard()
       .getBoardConfig()
       .getAcceptedLinks()
-      .map((link) => link.getName());
+      .map((l) => l.getName());
```

Comment:
pai da, sunt de acord, problema e ca e shadow variable. Tot link se cheama si parametrul trimis in functie ca argument. si devine neclar la care link se refera functia de map